artifacts/rhdh-plugins-reference/quickstart/quickstart-plugin-user.adoc

```diff
@@ -0,0 +1,72 @@
+[id="using-quickstart-plugin"]
+= Using the Quickstart plugin
+
+The Quickstart plugin provides a guided onboarding experience for new users of {product}. It displays a customizable drawer interface with interactive quickstart steps that help users get familiar with the platform.
+
+== Features
+Interactive Drawer Interface:: Displays quickstart steps in a slide-out drawer.
+Progress Tracking:: Tracks completion status of individual steps with persistent storage.
+Configurable Content:: Define custom quickstart items through app configuration.
+Visual Progress Indicator:: Shows overall completion progress with a progress bar.
+Call-to-Action Support:: Each step can include clickable action buttons.
+
+.Procedure
+. Open the Quickstart drawer
+.. Navigate to the **top right corner** of the {product-short} interface.
+.. Click the icon labeled **“Quick start”** (represented by a question mark icon in the top menu).
```

Comment:
Its labeled (has a tooltip) as Help. Clicking upon that opens the dropdown which has a quickstart menu-item.